python/dask_cudf/setup.py

```diff
@@ -8,8 +8,14 @@
 
 import versioneer
 
+cudf_versioned = (
+    "cudf "
+    f"{os.environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').lstrip('v')}"
+    f"{os.environ.get('VERSION_SUFFIX', '')}"
+)
```

Comment:
🤔 This isn't very friendly to source installs, since it relies on CI environment variables. For example, I don't think you could install a cudf package from conda and install dask-cudf from source with this construction.